send.go

```diff
@@ -127,6 +127,11 @@ func (d *DBFT) sendCommit() {
 }
 
 func (d *DBFT) sendRecoveryRequest() {
+	// If we're here, something is wrong, we either missing some messages or
+	// transactions or both, so re-request missing transactions here too.
+	if d.RequestSentOrReceived() && !d.hasAllTransactions() {
+		d.processMissingTx()
```

Comment:
I think this is part of the user of this library to schedule and re-request missing tx until they are received/
But nothing is wrong with requesting them twice if we need it.